src/view/components/common/login-component/login-component.ts

```diff
@@ -0,0 +1,201 @@
+import {Component, Input, Inject, EventEmitter, Output, ViewEncapsulation} from '@angular/core';
+import {MD_INPUT_DIRECTIVES} from '@angular2-material/input/input';
+import {MdButton} from '@angular2-material/button';
+import {MdToolbar} from '@angular2-material/toolbar';
+import {MdCheckbox} from '@angular2-material/checkbox/checkbox';
+import {MD_CARD_DIRECTIVES} from '@angular2-material/card';
+import {LoginService} from '../../../../api/services/login-service';
+
+@Component({
+    directives: [MdToolbar, MD_INPUT_DIRECTIVES, MdButton, MdCheckbox, MD_CARD_DIRECTIVES],
+    encapsulation: ViewEncapsulation.Emulated,
+    moduleId: __moduleName, // REQUIRED to use relative path in styleUrls
+    providers: [LoginService],
+    selector: 'dot-login-component',
+    styleUrls: ['login-component.css'],
+    templateUrl: ['login-component.html'],
+})
+
+/**
+ * The login component allows the user to fill all
+ * the info required to log in the dotCMS angular backend
+ */
+export class LoginComponent {
+    @Input() my_account_login: string= '@dotcms.com';
```

Comment:
According to [Microsoft](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines) we should use camelCase to name variables, same principle apply for Javascript code.